sass/components/_chart-controls.scss

```diff
@@ -26,9 +26,30 @@
             transform: rotate(225deg); /* 180 + 45 */
             display: none;
         }
+        .cq-menu-btn{
+            position: relative;
+        }
         .cq-menu-dropdown {
             height: auto;
         }
+        .notification-badget {
```

Comment:
@shayanbinary. Pls. I said "notification badge". Not badget.

Reply:
ooops